deployments/workloads.go

```diff
@@ -229,15 +246,45 @@ func (w Workloads) createGiteaCredsSecret(c *kubernetes.Cluster, options kuberne
 }
 
 func (w Workloads) createWorkloadsServiceAccountWithSecretAccess(c *kubernetes.Cluster) error {
-	automountServiceAccountToken := false
+	automountServiceAccountToken := true
```

Comment:
Why is this change needed now ? We need the service account token in the workload pods only if the pod containers interact with the k8s API. IIRC, we do that in the predictor image, but then how did that ever work ?

Reply:
The reason it is working now is because the workloads pods have been using the `staging-triggers-admin` service account, not the workloads SA.

Reply:
ok, makes sense, thank you !